admin/test/indexes/TagPagesTest.scala

```diff
@@ -39,34 +49,42 @@ import test.WithTestExecutionContext
     )
 
     for ((unicode, ascii) <- words) {
-      tagPages.alphaIndexKey(unicode) shouldEqual ascii
+      alphaIndexKey(unicode) shouldEqual ascii
     }
   }
 
-  it should "index by 0-9 if the first character is a digit" in {
```

Comment:
Note that the test _description_ says '0-9', but we actually _use_ '1-9' - consequently this page exists:

https://www.theguardian.com/index/contributors/1-9

...we maybe don't want to change the url at this point, even if '0-9' would be numerically more correct?!?

Reply:
👌